contracts/ERC677Receiver.sol

```diff
@@ -0,0 +1,6 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.6.10;
+
+interface ERC677Receiver {
```

Comment:
Do we want the interfaces in separate files or in `WETH10.sol`.

I'm in favour that when they are one-liners we should follow the MakerDAO pattern and add them in `WETH.sol`.

Reply:
I've moved it to the top of WETH.sol